test/source/tests/setup.ts

```diff
@@ -444,6 +444,8 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
       await composeFrame.waitAndClick('@action-send');
       await inboxPage.waitAll('@dialog-passphrase');
       const passphraseDialog = await inboxPage.getFrame(['passphrase.htm']);
+      await Util.sleep(5);
+      expect(await passphraseDialog.readHtml('@lost-pass-phrase-with-ekm')).to.equal('Ask your IT staff for help if you lost your pass phrase.');
```

Comment:
Please use `waitForContent` method, which will keep retrying until it can find the text, instead of wait + try once. (it will execute faster, as it doesn't always have to wait for 5 seconds)

Otherwise this looks good. You should also alter one more test - a test that doesn't use EKM, to test the opposite scenario. I'll highlight the discussion below.